scarpe-components/lib/scarpe/components/calzini/art_widgets.rb

```diff
@@ -57,6 +57,33 @@ def star_element(props, &block)
     end
   end
 
+  def oval_element(props, &block)
+    dc = props["draw_context"] || {}
+    fill = dc["fill"]
+    stroke = dc["stroke"]
```

Comment:
i think it's as u r accessing stroke from drawing context but passing to props..

this will give stroke in a scenario where we do

```
Shoes.app do
stroke red
oval x x x
end
```

Reply:
Yup, this. Probably makes sense to do something like 'stroke = props["stroke"] || dc["stroke"] || "black"'

Basically use an explicit one in the oval constructor if there is one, fall back to using draw context, fall back to default.